tests/llm/fixtures/test_ask_holmes/09_crashpod/test_case.yaml

```diff
@@ -51,3 +51,4 @@ after_test: |
 tags:
   - easy
   - kubernetes
+  - one-test
```

Comment:
what is one-test tag?

Reply:
to run a benchmark on tag one-test that is reliable fast single test that passes to verify the benchmarks/models is set up correctly.
documented in notion for our benchmarks
see more details here
https://github.com/HolmesGPT/holmesgpt/pull/1137/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711